src/lib/SSVKeys.ts

```diff
@@ -152,7 +149,7 @@ export class SSVKeys {
     ) {
       throw Error('Operator public keys and shares public/encrypted keys length does not match or have zero length.');
     }
-    this.keySharesInstance.generateContractPayload({
+    this.keyShares.generateContractPayload({
```

Comment:
make `generateContractPayload` returning payload so that it would be faster to do:
`return this.keyShares.generateContractPayload(...)`